portality/templates/doaj/article.html

```diff
@@ -61,16 +63,21 @@
     <div class="span4">
         <div class="box with-borders" style="height: 120px">
             {% if article.bibjson().journal_title %}
+                {% if article.bibjson().get_identifiers(idtype='pissn') or article.bibjson().get_identifiers(idtype='eissn') %}
+                    <p><strong>Journal</strong>:&nbsp;
+                    <a href={{ "/toc/" + article.bibjson().get_identifiers(idtype='pissn')[0] or article.bibjson().get_identifiers(idtype='eissn')[0]}}>{{ article.bibjson().journal_title }}</a></p>
```

Comment:
Well I think it worked when I tested it! Feel free to check out the branch first and have a go, or ask me to provide proof.